package.json

```diff
@@ -7,6 +7,7 @@
     "format:write": "yarn prettier --write .",
     "lint": "echo 'Linting all packages' && wsrun --parallel --exclude-missing lint",
     "test": "echo 'Running tests' && wsrun --parallel --exclude-missing test",
+    "build": "echo 'Building all packages' && wsrun --parallel --exclude-missing build",
```

Comment:
Don't run build in parallel ... run in `--stages`, it will pickup package with no internal dependencies. Otherwise dependent package will break

Reply:
🧠 totally!